cluster/cluster.yaml

```diff
@@ -2337,8 +2335,8 @@ Resources:
             Version: 2012-10-17
           PolicyName: root
       RoleName: "{{.Cluster.LocalID}}-etcd-backup"
-{{- end }}
     Type: 'AWS::IAM::Role'
+{{- end }}
```

Comment:
@tcondeixa this misplaced conditional end was responsible for the duplicated `Type: 'AWS::IAM::Role'` in the template :)